### ISSUE TITLE
[spiegel] Some URLs aren't extracted properly anymore

### ISSUE THREAD

**Issue by eliasp** [rg3/youtube-dl#4579]
The URL following URL:
`http://www.spiegel.de/video/neu-im-kino-mit-exodus-honig-im-kopf-und-entdeckung-der-unendlichkeit-video-1543886.html`
should return the URL:
`http://video2.spiegel.de/flash/68/83/1543886_1024x576_H264_HQ.mp4`
but returns instead:
`http://video2.spiegel.de/flash/68/83/1543886_1920x1080_HD.mp4`

The document source contains this JS variables which contains some of the information required to stitch the correct URL together:
```JS
                var credit                      = "SPIEGEL ONLINE";
                var displaycat          = "5";
                var videocat            = "44";
                var trackingsize        = "gross";
                var startmod            = "true";
                var server                      = "http://video2.spiegel.de/flash/68/83/";
                var embedWidth          = 640;
                var embedHeight         = 360;
```

The call to the `spStartVideo5()` function contains the remaining stuff:
```JS
spStartVideo5(1543886, '1543886_1024x576_H264_HQ.mp4', 'oldVideo3', poster, prerollGateway, postrollGateway, 1, credit, displaycat, videocat, server, trackingsize, startmod, 'onAdStart', 'onAdEnd', 'onVideoEnd', 'onFinalEnd', embedWidth, embedHeight);
```

**Comment by phihag** [comment]
You're simply citing format 19, while youtube-dl picks format 26. The question is why that format is mentioned in [Spiegel's index file](http://video2.spiegel.de/flash/68/83/1543886.xml), but not actually available.

**Comment by dstftw** [comment]
This issue will be fixed in the next version. Thanks for the report.

**Comment by phihag** [comment]
Fixed in youtube-dl 2015.01.15.1 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
